homework/index.js

```diff
@@ -29,8 +29,42 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo) {
+    const reposList = document.getElementById('reposList');
+    const repoItem = createAndAppend('li', reposList);
+    const commonBlock = createAndAppend('div', repoItem, {
+      class: 'commonBlock',
+    });
+    const dataTypeList = createAndAppend('ul', commonBlock, {
+      class: 'dataTypeList',
+    });
+    createAndAppend('li', dataTypeList, { text: 'Repository:' });
+    createAndAppend('li', dataTypeList, { text: 'Description:' });
+    createAndAppend('li', dataTypeList, { text: 'Forks:' });
+    createAndAppend('li', dataTypeList, { text: 'Updated:' });
+    const dataList = createAndAppend('ul', commonBlock, { class: 'dataList' });
+    const repoLink = createAndAppend('li', dataList);
+    createAndAppend('a', repoLink, {
+      href: repo.html_url,
+      target: '_blank',
+      class: 'app-link',
+      text: repo.name,
+    });
+    if (repo.description === null) {
+      createAndAppend('li', dataList, {
+        text: 'No Description',
+        class: 'alert-no-description',
+      });
+    } else {
+      createAndAppend('li', dataList, { text: repo.description });
+    }
+    createAndAppend('li', dataList, { text: repo.forks });
+    createAndAppend('li', dataList, { text: repo.updated_at });
```

Comment:
Can you format this date in a more user-friendly format?

Reply:
Done